src/Bicep.Decompiler/BicepHelpers/SyntaxHelpers.cs

```diff
@@ -98,6 +98,7 @@ public static class SyntaxHelpers
             ["greater"] = TokenType.GreaterThan,
             ["greaterOrEquals"] = TokenType.GreaterThanOrEqual,
             ["equals"] = TokenType.Equals,
+            ["notEquals"] = TokenType.NotEquals,
```

Comment:
Could you add a comment over this to explain that it's not a 'real' ARM template function (e.g. you're just using it to simplify the conversion)?